pkgs/development/coq-modules/multinomials/default.nix

```diff
@@ -1,18 +1,22 @@
 { coq, mkCoqDerivation, mathcomp, mathcomp-finmap, mathcomp-bigenough,
-  lib, version ? null }:
+  lib, version ? null, useDune ? false }@args:
 with lib; mkCoqDerivation {
 
   namePrefix = [ "coq" "mathcomp" ];
   pname = "multinomials";
   owner = "math-comp";
   inherit version;
   defaultVersion =  with versions; switch [ coq.version mathcomp.version ] [
+      { cases = [ (range "8.10" "8.13") "1.12.0" ];             out = "1.5.4"; }
+      { cases = [ (range "8.10" "8.12") "1.12.0" ];             out = "1.5.3"; }
```

Comment:
although it could also be considered as a documentation for the compatibility of 1.5.3 releases.
Moreover if I reified the list (by moving the call to `switch` to the `mkCoqDerivation`) it could also act as an actual compatibility checker... So... as dead codes go, I'd say this one is more relevant than usual...